### ISSUE TITLE
goto jumps to top of file, when seemingly the symbol is unknown

### ISSUE THREAD

**Issue by joprice** [reasonml-editor/reasonml-idea-plugin#258]
plugin version: 0.89.0-2020.1

### Description

When using the Goto action (cmd+b on mac), I am able to jump to local definitions, but when I attempt to jump to a library function, the cursor jumps to the top of the file, and I lose my position. If I try to jump to an implementation however, I get the error message "No implementations found" and the cursor location is maintained. I would expect a similar popover error to be shown in both cases.

**Comment by giraud** [comment]
Hi, thanks for the report, but unfortunately this is too generic.

**Comment by joprice** [comment]
What would help to clarify it?

**Comment by giraud** [comment]
I believe there are already issues that cover your problem

**Comment by joprice** [comment]
I see. I did search around and found one related to interface/implementation issues, but not to unresolved symbols from other files. I thought this was more specific as I'm not expecting symbols to be resolved, but just to not lose position and show a relevant error. I assumed it would be something where a different return type or exception handling would fix it. I did poke around the code, but I'm unfamiliar with the intellij plugin apis. Feel free to close if it's not useful to track separately.
